pkg/types/kustomization.go

```diff
@@ -79,6 +79,9 @@ type Kustomization struct {
 
 	// ImageTags is a list of ImageTag for changing image tags
 	ImageTags []ImageTag `json:"imageTags,omitempty" yaml:"imageTags,omitempty"`
+
+	// JSONPatches is a list of JSONPatch for applying json patch
+	JSONPatches []JSONPatch `json:"jsonPatch,omitempty" yaml:"jsonPatch,omitempty"`
 }
```

Comment:
Before submitting this, should add some code that detects the field and
log.Print("Field ignored; no implementation yet.")